packages/utils/src/test-utils/getForcProject.ts

```diff
@@ -7,22 +7,24 @@ import { normalizeString } from '../utils/normalizeString';
 interface IGetForcProjectParams {
   projectDir: string;
   projectName: string;
+  build: 'debug' | 'release';
```

Comment:
I added this to allow support for both `debug` and `release` builds with this utility. This might be overengineering because for our purposes `release` is enough. The train of thought is that we're exporting this utility to our users so I wanted to not break their code and they might sometimes want to compile to debug as well.